<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGeneral</class>
 <widget class="QWidget" name="ConfigureGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="toggle_deepscan">
            <property name="text">
             <string>Recursive scan for game folder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="toggle_check_exit">
            <property name="text">
             <string>Confirm exit while emulation is running</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
      <item>
        <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
            <string>Performance</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                  <widget class="QCheckBox" name="toggle_cpu_jit">
                    <property name="text">
                      <string>Enable CPU JIT</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Emulation</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Region:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="region_combobox">
              <item>
               <property name="text">
                <string notr="true">JPN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">USA</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">EUR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">AUS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">CHN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">KOR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">TWN</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Hotkeys</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="GHotkeysDialog" name="widget" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GHotkeysDialog</class>
   <extends>QWidget</extends>
   <header>hotkeys.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
